src/main/java/org/jitsi/protocol/xmpp/AbstractOperationSetJingle.java

```diff
@@ -262,21 +262,26 @@ public void sendAddSourceIQ(MediaSourceMap ssrcs,
                                 MediaSourceGroupMap ssrcGroupMap,
                                 JingleSession        session)
     {
-        JingleIQ addSourceIq
-            = new JingleIQ(JingleAction.SOURCEADD, session.getSessionID());
+        boolean onlyInjected =
+                ssrcs.getSourcesForMedia("video").isEmpty() &&
+                        ssrcs.getSourcesForMedia("audio").stream().allMatch(SourcePacketExtension::isInjected);
+        if (onlyInjected)
+        {
+            logger.info("Suppressing source-remove for injected SSRC.");
```

Comment:
Do we really want an INFO-level log for every one of these suppressions?  Isn't the whole reason we're doing this that there're way too many of them?